subprojects/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/KotlinDslJvmDefaultIntegrationTest.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.kotlin.dsl.integration
+
+import org.gradle.integtests.fixtures.RepoScriptBlockUtil.mavenCentralRepository
+import org.gradle.test.fixtures.dsl.GradleDsl.KOTLIN
+import org.junit.Test
+
+
+class KotlinDslJvmDefaultIntegrationTest : AbstractPluginIntegrationTest() {
+
+    @Test
+    fun `kotlin-dsl java and groovy consumers can use kotlin interface default methods directly`() {
```

Comment:
❌ Nice! We should also have a test for a Kotlin DSL script consumer implementing a Java interface with default methods.